les/client.go

```diff
@@ -18,6 +18,7 @@
 package les
 
 import (
+	"context"
```

Comment:
Are people OK with this addition? I'm naive, but it feels a bit odd to me that we need a whole new import that upstream did not.

Reply:
It's because you're trying to do a `rpc.DialContext`. This is a totally fine import. The only way to not have this import is to not use the context which is way worse.

Reply:
It's also part of the standard library